### PR TITLE
Cleaned up PyAibyte docs (PR # 35603)

### LINKED ISSUE #36: Docs: Remove 'Central DB' from diagram
The diagram shows 'Central DB', which isn't really part of the model. Everything is represented by the SQLCache node, which can either be local or remote.

Image

### PR DESCRIPTION
**Pull request by bindipankhudi**
## What
Resolves: https://github.com/airbytehq/PyAirbyte/issues/36 & part of https://github.com/airbytehq/PyAirbyte/issues/55

## How
Made following changes to the PyAirbyte docs 
- Replaced instances of AirbyteLib with PyAirbyte on getting started page
- Updated architecture image 
- Added a PyAirbyte logo 
- Added Quickstarts section
- Removed reference.mdx
- Added redirect from /pyairbyte to /using-airbytelib/airbytelib/getting-started

### CODE REVIEW

File: docs/using-airbyte/airbyte-lib/getting-started.mdx [hunk 1]
```diff
@@ -38,17 +38,26 @@ for name, records in result.streams.items():
     print(f"Stream {name}: {len(list(records))} records")
 ```
 
+## Quickstarts
+
+* [CoinAPI](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_CoinAPI_Demo.ipynb)
+* [GA4](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_GA4_Demo.ipynb) 
+* [Shopify](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/PyAirbyte_Shopify_Demo.ipynb) 
+* [Github](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_Github_Incremental_Demo.ipynb) 
+* [PostGres (cache)](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/PyAirbyte_Postgres_Custom_Cache_Demo.ipynb)
```
Comment (aaronsteers) [inline docs/using-airbyte/airbyte-lib/getting-started.mdx:47]:
```suggestion
* [Postgres (cache)](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/PyAirbyte_Postgres_Custom_Cache_Demo.ipynb)  
```

File: docs/using-airbyte/airbyte-lib/getting-started.mdx [hunk 2]
```diff
@@ -38,17 +38,26 @@ for name, records in result.streams.items():
     print(f"Stream {name}: {len(list(records))} records")
 ```
 
+## Quickstarts
+
+* [CoinAPI](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_CoinAPI_Demo.ipynb)
+* [GA4](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_GA4_Demo.ipynb) 
+* [Shopify](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/PyAirbyte_Shopify_Demo.ipynb) 
+* [Github](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_Github_Incremental_Demo.ipynb)
```
Comment (aaronsteers) [inline docs/using-airbyte/airbyte-lib/getting-started.mdx:46]:
```suggestion
* [GitHub](https://github.com/airbytehq/quickstarts/blob/main/airbyte_lib_notebooks/AirbyteLib_Github_Incremental_Demo.ipynb) 
```

**Review by aaronsteers — CHANGES_REQUESTED** [review]
Small nits. Will apply those and then approve.

**Review by aaronsteers — APPROVED** [review]
Looks great - thanks, @bindipankhudi ! 


Approving, and expecting this will auto-merge after CI turns green again.

**Comment by bindipankhudi** [comment]
Thanks for merging! Didn’t mean for u to make the changes :)

**Pull request merged**